Gemfile

```diff
@@ -43,7 +43,7 @@ group :system_tests do
   if beaker_version = ENV['BEAKER_VERSION']
     gem 'beaker', *location_for(beaker_version)
   else
-    gem 'beaker', '>= 3.9.0', :require => false
+    gem 'beaker', '>= 4.2.0', :require => false
```

Comment:
can you please rebase against our master? Since https://github.com/voxpupuli/puppet-splunk/pull/211 got merged, those changes to the Gemfile shouldn't be needed.